plugin.cpp

```diff
@@ -248,6 +253,55 @@ PLUGIN_HANDLE plugin_init(ConfigCategory* config,
 	return (PLUGIN_HANDLE)info;
 }
 
+
+/**
+ * Flatten nested datapoint
+ *
+ * @param datapoint	datapoint to flatten
+ * @param assetName	assetname to which datapoint belongs
+ * @param newReadings	vector for newreadings of flatten datapoint
+ */
+void flattenDatapoint(Datapoint *datapoint, std::string assetName, vector<Reading *>& newReadings)
```

Comment:
You can drop all the reference to `std::` as there must be a `using namespace std;` above, otherwise vector would not work.